ethers-solc/src/compile.rs

```diff
@@ -162,7 +166,8 @@ impl Solc {
         // load the local / remote versions
         let versions = svm::installed_versions().unwrap_or_default();
         let local_versions = Self::find_matching_installation(&versions, &sol_version);
-        let remote_versions = Self::find_matching_installation(&RELEASES, &sol_version);
+        let (_releases, sorted_releases) = RELEASES.clone();
+        let remote_versions = Self::find_matching_installation(&sorted_releases, &sol_version);
```

Comment:
Do we need the `clone`? Since it's by reference, can't we do `&RELEASES.1` or something instead of `&sorted_releases`?

Reply:
Yea that sounds more reasonable.